src/runtime/Python.Runtime.csproj

```diff
@@ -51,6 +51,7 @@
     <PlatformTarget>x64</PlatformTarget>
     <CodeAnalysisIgnoreBuiltInRuleSets>false</CodeAnalysisIgnoreBuiltInRuleSets>
     <CodeAnalysisIgnoreBuiltInRules>true</CodeAnalysisIgnoreBuiltInRules>
+    <DefineConstants>PYTHON35, UCS2</DefineConstants>

```

Comment:
This needs to be guarded with a condition on DefineConstants not being already set. Otherwise the ReleaseWin|x64 target will always try and build the Python 3.5 build, regardless of the settings passed in via setup.py.

This line can probably just be removed (see line 47 just above here).